src/app.ts

```diff
@@ -49,6 +49,7 @@ export async function app(localVars: any = {}) {
     port: configuration.apiPort,
     dbUrl: configuration.mongodbUrl,
     rabbitmqUrl: configuration.rabbitmqUrl,
+    ipfsUrl: configuration.ipfsUrl,
```

Comment:
API shouldn't need to use IPFS - it's the HealthModule's responsibility to collect health data

Reply:
Ok. So we would want the health module to check for IPFS connectivity and then store it in mongo to be read by the API module?

Reply:
Yep, your initial approach was right about this. 

And yeah, no problem doing that refactor in another PR.